Add a simple statsClient object

eatyourgreens opened the pull request:
Set project and workflow via statsClient.update().
Get stats with statsClient.statCount(period, type).

brian-c commented on lib/stats-client.js:
```diff
@@ -0,0 +1,35 @@
+  },
+
+  statCount: function (period, type) {
+    var query = this.workflowId ? "workflow_id=" + this.workflowId : "project_id=" + this.projectId;
```
Why store workflow ID and project ID as properties on the client instead of passing them into the function? Makes it kind of unnecessarily stateful, right?

brian-c commented on lib/stats-client.js:
```diff
@@ -0,0 +1,35 @@
+            return results["events_over_time"]["buckets"]
+          }
+        )
+        .catch(
```
I'd leave any errors to the consumer to handle. As written, this can only fail silently to the end user.

eatyourgreens commented:
You're right. I've rebased and switched to a single query method that takes the workflow ID and project ID as parameters. Should reject on errors, and it's up to the client what to do in that case.

brian-c commented:
Nice, thanks!
